examples/http-request-error-burnrate.yaml

```diff
@@ -1,166 +1,106 @@
 alerts:
 - alert: ErrorBudgetBurn
   annotations:
-    message: 'High requests error budget burn for namespace=default,job=fooapp (current value: {{ $value }})'
+    message: 'High error budget burn for namespace=default,job=fooapp (current value: {{ $value }})'
   expr: |
-    (
-      status_class_5xx:promhttp_metric_handler_requests_total:ratio_rate1h{namespace="default",job="fooapp"} > (14.4*0.001000)
-      and
-      status_class_5xx:promhttp_metric_handler_requests_total:ratio_rate5m{namespace="default",job="fooapp"} > (14.4*0.001000)
-    )
-    or
-    (
-      status_class_5xx:promhttp_metric_handler_requests_total:ratio_rate6h{namespace="default",job="fooapp"} > (6*0.001000)
-      and
-      status_class_5xx:promhttp_metric_handler_requests_total:ratio_rate30m{namespace="default",job="fooapp"} > (6*0.001000)
-    )
+    sum(promhttp_metric_handler_requests_total:burnrate5m{namespace="default",job="fooapp"}) > (14.40 * (1-0.99900))
```

Comment:
the selectors are already included with the recording rule, why do you need to add them here as well?

Reply:
We need to have them here, as we otherwise may sum `promhttp_metric_handler_requests_total:burnrate5m` across multiple namespaces, as an example. We only want to sum for the specific use case and only alert in that case.